### PR TITLE
Badge

### PR DESCRIPTION
**Pull request by reubenjnazareth**

### CODE REVIEW

File: badge.html [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+</div>
+
+<div class="component-title">BADGE</div>
+<div class="component-intro">A badge is used to display information such as online or ofline or busy. Badges are also used to display information to user with a quick glace by showing a number on the icon. For example, a badge on the inbox icon could depict the count of the number of unread messages.</div>
+
+
+<div class="sub-title">Badge on icons</div>
+<div class="badge-nav">
+
+
```
Comment (chanchal16) [inline badge.html]: remove these blank spaces

**Review by chanchal16 — APPROVED** [review]

File: badge.html (hunk 1 above)
Comment (reubenjnazareth) [reply badge.html]: corrected!

File: badge.html [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  sandbox="allow-scripts allow-same-origin">
+</iframe>
+
+<hr class="line">
+
+<div class="sub-title">Badge on Avatar</div>
+<div class="badge-nav">
+
+    
+
```
Comment (sharath-io) [inline badge.html:79]: Format the code to remove blank spaces.

**Review by sharath-io — APPROVED** [review]
Format the code to remove blank spaces.
use variable names for colors, if you are re-using them.

**Pull request merged**